Docs/configuration user view (#162)
CombiRWTH/StaffScheduling: docs/user-view/configuration/staff.md

> Currently, our application lacks a user-friendly interface for comfortably managing configurations. Instead, these settings are stored in a JSON file, which requires manual editing to make any changes.

Reviewer: Shouldn't this be part of the index? This is a general information not specific to the employees.

Reviewer: Or maybe put it inside an info box like https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types

### Adding or Deleting Employees

Employee information is stored in the file located at `cases/{case_id}/employees.json`. Each employee entry follows this structure:

```json
{
    "firstname": "Annelene",
    "key": 6928,
    "name": "Izzo",
    "type": "Medizinische/r Fachangestellte/r (81102-004)"
}
```

> To **add an employee**, create a new JSON object with the appropriate fields for each employee you wish to include. Ensure that you assign a unique key for each employee, as this is essential for identification within the system.

Reviewer: ```suggestion
To **add an employee**, create a new JSON object containing the required fields for each employee. Assign a unique key to each employee, as this is essential for identification within the system.
```
